documentation/docs/services/audit-trail-gateway/use-cases.md

```diff
@@ -0,0 +1,60 @@
+---
+image: /img/integration-services/diagrams/ecommerce-audit-trail-bridge.jpeg
+description: In the context of e-commerce and the ENSURESEC project, you can use the Audit Trail Gateway to share immutable and auditable data and events and guarantee the Immutability of Large Data Sets.
+keywords:
+- API definition
+- auditable data
+- immutability
+- large data set
+- writer subscriber
+- commerce infrastructure
+- audit trail hash
+---
+
+# Use Cases
+
+In the context of e-commerce and the [ENSURESEC](https://www.ensuresec.eu/) project, you can use the Audit Trail Gateway
+to share immutable and auditable data and events and guarantee the Immutability of Large Data Sets.
+
+## Sharing Immutable and Auditable Data and Events
+
+This scenario involves a small amount of data that a channel *Author* and *writer subscribers* want to share with *
+reader subscribers*. All the information is stored in the Audit Trail, and channel reader subscribers can verify its
+source (identity of the writer subscribers).
+
+One example of this scenario is _sharing threat information_ detected by a sensor on a critical e-commerce
+infrastructure to alert all other systems connected to the same infrastructure.
+
+You could implement the following workflow:
+
+1. **Company X** has [registered and verified](../SSI-bridge/API-definition) its [Identity](../../basics/identity.md) as
+   well as an identity for **employee Y** and **device (sensor) Z**.
+2. **Company A** has been registered and verified its identity as well as the identity of an **employee B** and **tool
+   C**.
+3. The **device Z** uses the [Gateway(GW) APIs](API-definition.md) to [create **
+   channel**](API-definition.md#channel-service) **A**.
+4. The **tool C** uses the GW APIs to [search for channels](API-definition.md#channel-info-service) based on available
+   indexing metadata and [request a subscription](API-definition.md#subscription-service) to the selected **channel A**.
+5. The **device Z** uses the GW APIs to authorize **tool C** to access the **channel A** as a reader.
+6. The **device Z** uses the GW APIs to add data to **channel A**.
+7. **Tool C** is automatically notified of new data coming from **device Z**.
+
+## Guaranteeing the Immutability of Large Data Sets
+
+This scenario includes storing and sharing data across an organizations' data sets maintained in large data lakes while
+guaranteeing that the data sets have not been altered over time or passed to unauthorized parties.
```

Comment:
The data is used both as a qualifier and a subject. I just found it a bit confusing.

```suggestion
This scenario includes storing and sharing information across an organizations' data sets maintained in large data lakes while
guaranteeing that the data sets have not been altered over time or passed to unauthorized parties.
```